build

```diff
@@ -9,7 +9,7 @@ if [ ! -h gopath/src/${REPO_PATH} ]; then
 fi
 
 export GOBIN=${PWD}/bin
-export GOPATH=${PWD}/gopath
+export GOPATH=${PWD}/gopath:`pwd`/Godeps/_workspace
```

Comment:
I misread slightly, thought it was including predefined GOPATH. In this
case, either option seems equally unintuitive to me, so this is fine :P

On Fri, Jun 12, 2015 at 12:59 PM, Eugene Yakubovich <
notifications@github.com> wrote:

> In build <https://github.com/appc/cni/pull/21#discussion_r32349501>:
>
> > @@ -9,7 +9,7 @@ if [ ! -h gopath/src/${REPO_PATH} ]; then
> >  fi
> >
> >  export GOBIN=${PWD}/bin
> > -export GOPATH=${PWD}/gopath
> > +export GOPATH=${PWD}/gopath:`pwd`/Godeps/_workspace
>
> Why Godeps first? Wouldn't you want code local to this repo to be searched
> before dependencies?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appc/cni/pull/21/files#r32349501>.
>